### ISSUE TITLE
Jablotron integration stopped working

### ISSUE THREAD

**Issue by Whitenoises** [kukulich/home-assistant-jablotron100#60]
Since 1 day the integration stopped working after it was working for one year.
I'm getting these errors:

2022-12-23 14:11:48.711 WARNING (SyncWorker_1) [homeassistant.loader] We found a custom integration jablotron100 which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2022-12-23 14:11:59.738 INFO (MainThread) [homeassistant.bootstrap] Setting up stage 2: {'system_health', 'history', 'input_datetime', 'blueprint', 'light', 'logbook', 'schedule', 'zone', 'image', 'group', 'sun', 'homekit', 'trace', 'input_text', 'persistent_notification', 'ring', 'input_button', 'analytics', 'rpi_power', 'upnp', 'doorbird', 'ffmpeg', 'file_upload', 'lovelace', 'hardware', 'scene', 'repairs', 'automation', 'overkiz', 'diagnostics', 'shelly', 'homeassistant_alerts', 'hacs', 'default_config', 'hue', 'met', 'input_select', 'my', 'utility_meter', 'mqtt', 'tts', 'jablotron100', 'raspberry_pi', 'dlna_dmr', 'deconz', 'tag', 'energy', 'ipp', 'device_automation', 'input_boolean', 'mobile_app', 'onboarding', 'counter', 'timer', 'search', 'map', 'vicare', 'script', 'person', 'auth', 'application_credentials', 'config', 'media_source', 'input_number'}
2022-12-23 14:11:59.857 INFO (MainThread) [homeassistant.setup] Setting up jablotron100
2022-12-23 14:11:59.858 INFO (MainThread) [homeassistant.setup] Setup of domain jablotron100 took 0.0 seconds
2022-12-23 14:12:00.013 ERROR (SyncWorker_3) [homeassistant.util.json] Could not parse JSON content: /config/.storage/jablotron100
2022-12-23 14:12:00.245 ERROR (MainThread) [homeassistant.config_entries] Error setting up entry Jablotron for jablotron100
File "/config/custom_components/jablotron100/__init__.py", line 22, in async_setup_entry
File "/config/custom_components/jablotron100/jablotron.py", line 268, in initialize
File "/config/custom_components/jablotron100/jablotron.py", line 370, in _load_stored_data
2022-12-23 14:12:00.522 DEBUG (MainThread) [homeassistant.core] Bus:Handling <Event component_loaded[L]: component=jablotron100>

PS: I downgraded Jablotron as I Was using it in an older version but nothing helps...
I also removed and reistalled HACS

![Screenshot](https://user-images.githubusercontent.com/91630346/209343071-ac5790ec-c02a-460c-ac5b-b6ec67172076.JPG)

**Comment by kukulich** [comment]
Try to remove the file `/config/.storage/jablotron100`.

**Comment by Whitenoises** [comment]
I just went back to a backup I did 2 Weeks ago and it works. Now I am afraid to update hacks and Jablotron.
Also, when I removed Hacs before, I also deleted the files in the "custom_components" folders of Hacs. I could not delete the folder because I received an "errno-39-directory-not-empty" error. I did the same for Jsblotron.
(I guess by now you must have realized that I am a complete noob when it comes to Linux...)
Will it be save to update Hassio and Jablotron?

**Comment by Whitenoises** [comment]
@kukulich In my hurry I forgot to say Thank you for your reply! Thank you!

**Comment by kukulich** [comment]
@Whitenoises I use latest HA and everything works so I think you should upgrade.

**Comment by Whitenoises** [comment]
@kukulich
I really appreciate your help. I have updated my Homeassistant after I did the backup two weeks ago and it works, but I will not update Hacs and Jablotron anymore because I am afraid it will not work again (see screenshot).
Will there ever be an official Integation for Jablotron alarm systems?
Thank you and Happy Holidays,
![Working Jablotron](https://user-images.githubusercontent.com/91630346/209349091-3db83c23-5825-4403-88bf-f7f2345c36cd.JPG)
